extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/common/KoofrClient.java

```diff
@@ -102,13 +103,9 @@ public boolean fileExists(String path) throws IOException, InvalidTokenException
       if (code == 404) {
         return false;
       }
-      throw new IOException(
-          "Got error code: "
-              + code
-              + " message: "
-              + response.message()
-              + " body: "
-              + response.body().string());
+      ResponseBody body = response.body();
+      String bodyAsString = body != null ? body.string(): null;
```

Comment:
if the above (updating method signature) is done, we can also update the test for this file: https://github.com/google/data-transfer-project/blob/master/extensions/data-transfer/portability-data-transfer-koofr/src/test/java/org/datatransferproject/transfer/koofr/common/KoofrClientTest.java#L75